docs/tutorial/image_det.md

```diff
@@ -1,3 +1,169 @@
 # Training for Object Detection
 
-Object detection is in preparation.
+This guide trains a neural network model to object detection of Human Face and Human Hand in the [Open Images Dataset V4](https://storage.googleapis.com/openimages/web/index.html) on GPU server.
+
+<img src="../_static/openimages_v4.png" width="600">
+
+## Preparation
+
+The Open Images V4 dataset is available from official website, but original dataset is too large for the tutorial. So, We provide reduced dataset.
+
+Blueoil supports 2 formats for object detection.
+
+- OpenImagev4 format
+- DeLTA-Mark format
+
+Note: *Please see the detail in <a href="../usage/dataset.html">Prepare training dataset</a>*
+
+You can download subset of Open Images V4 from
+[our server](https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/openimages.tgz).
+
+
+	$ wget https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/openimages.tgz
+	$ tar xf openimages.tgz
+
+
+This dataset consists of 5,000 color images in 2 classes, `Human face` and `Human hand`, with 2,500 images per class.
+
+## Generate a configuration file
+
+Generate your model configuration file interactively by running `blueoil init` command.
+
+    $ ./blueoil.sh init
+
+This is an example of initialization.
+
+```
+#### Generate config ####
+your model name ():  objectdetection
+choose task type  object_detection
+choose network  LMFYoloQuantize
+choose dataset format  OpenImagesV4
+training dataset path:  {dataset_dir}
+set validataion dataset? (if answer no, the dataset will be separated for training and validation by 9:1 ratio.)  no
+batch size (integer):  16
+image size (integer x integer):  160x160
+how many epochs do you run training (integer):  100
+initial learning rate:  0.1
+choose learning rate schedule ({epochs} is the number of training epochs you entered before):  '3-step-decay' -> learning rate decrease by 1/10 on {epochs}/3 and {epochs}*2/3 and {epochs}-1
+enable data augmentation?  No
+apply quantization at the first layer?  no
+```
+
+## Train a network model
+
+Train your model by running `blueoil train` command with model configuration. 
+
+    $ ./blueoil.sh train config/{Model name}.yml
+
+When training is started, training log and checkpoints are generated under `./saved/{Mode name}_{TIMESTAMP}` directory.
+
+Training is running on TensorFlow backend. So you can use TensorBoard to visualize your training process. 
```

Comment:
In order to user realize when the training finished, It is better to write how much MAP(Mean Average Presicion) are archived.

Reply:
I agree.